recommend/policy.go

```diff
@@ -77,9 +77,14 @@ func (img *ImageInfo) createPolicy(ms MatchSpec) (types.KubeArmorPolicy, error)
 
 	//	policy.Metadata["containername"] = img.RepoTags[0]
 	if ms.Name != "" {
-		policy.Metadata["name"] = ms.Name
+		policy.Metadata["name"] = strings.TrimPrefix(strings.TrimSpace(options.Usenamespace+"-"+mkPathFromTag(img.RepoTags[0])+"-"+ms.Name), "-")
```

Comment:
Declutter this line.

```suggestion
		policy.Metadata["name"] = strings.TrimPrefix(strings.TrimSpace(options.Usenamespace + "-" + mkPathFromTag(img.RepoTags[0]) + "-" + ms.Name), "-")
```
---
Also why we need to use `strings.TrimSpace()` here?